src/schema/portal/jgi_metagenomics.yaml

```diff
@@ -129,7 +129,7 @@ slots:
     name: dna_cont_well
     title: DNA well number
     comments:
-    - Required when 'plate' is selected for container type. Corner wells must be blank.
+    - Required when 'plate' is selected for container type. Corner wells must be blank/empty (A1, A12, H1, H12).
       For partial plates, fill by columns. Leave blank if the sample will be shipped
```

Comment:
It looks like there are two comments here. It looks like the hyphen `-` starting character wasn't used for the second comment.

Reply:
I will address the two comments above and add [the pattern from the issue](https://github.com/microbiomedata/issues/issues/15#issuecomment-1360480467)

Reply:
You have picked up so much about using GH for collaboration so quickly. Unfortunately we have a few poorly documented standards, like branch names. Please use this pattern:

> `issue-<number>-<brief keywords>`

Generally don't use capital letters or underscores. By convention, `issue-15-JGI-plate-layout` would have been a better branch name.

You can see other historical branch names at https://github.com/microbiomedata/nmdc-schema/branches/all